source/content/plugins-known-issues.md

```diff
@@ -1,4 +1,4 @@
----
+![CleanShot 2024-03-06 at 15 02 11@2x](https://github.com/pantheon-systems/documentation/assets/260253/cb95b35d-3dd4-4a24-89ad-c24316f79317)---
```

Comment:
@rachelwhitton should this screenshot be here?

Reply:
Yeah, It's in the wrong spot, but also I don't think we need this. We don't show a diff for the provided code changes anywhere else in this doc, let's not do it here. I removed it from the doc

Reply:
@stevector can you update your review and merge when ready? This is good to go on my side